src/features/surveys/managers/surveyResultsManager.ts

```diff
@@ -89,6 +92,18 @@ export const useSurveyResultsManager = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
+    const updateSurveyStatus = async () => {
+      await updateDoc(doc(db, 'surveys', surveyId), {
+        isActive: isSurveyActive,
+      });
```

Comment:
We should also handle errors here. When this action successed we should display `toast.success()` with success message and in other cases `toast.error()` with error message.